### ISSUE TITLE
Spanish lang stops understanding anything after a few messages

### ISSUE THREAD

**Issue by tobeorla** [wit-ai/wit#1384]
**Do you want to request a *feature*, report a *bug*, or ask a *question* about wit?** bug

**What is the current behavior?**

After  2 or 3 messages, (just repeating the same phrase `juego de tronos` => `game of thrones`), witai will always return `["result",{"_text":"","entities":{}}]`

The application is set in spanish lang.

I checked the socket and audio is begin send, so that part doesn't seem to be the problem.

![https://i.imgur.com/Txm5iZw.png](https://i.imgur.com/Txm5iZw.png)

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

Using official microphone sdk repeat the same word multiple times (waiting each time for server reply of course). After 2 o 3 tries it won't understand anything.

**What is the expected behavior?**

It should work as english does.

**What is the App ID where you are experiencing this issue (if applicable)?**

2231384743607802

**Comment by jtliao** [comment]
hey @tobeorla, are you just using the microphone sdk out of the box, or doing something else as well? just want to make sure the empty response is due to an issue on our end, rather than just empty audio

**Comment by tobeorla** [comment]
out of the box, since it is provided minified there is little margin to edit it.
If i reload the page, works fine again for 2 o 3 phrases, and then the same thing happens again.

**Comment by tobeorla** [comment]
https://github.com/wit-ai/wit/issues/1397 may be the same issue

**Comment by jtliao** [comment]
Does this still occur? We were experiencing some infra issues, not sure if it was related.

**Comment by tobeorla** [comment]
Yes it does, it didn't happen with english app while on spanish one did :o

**Comment by jtliao** [comment]
Can you give it a shot again? Having trouble reproducing this, as we've been having some sporadic problems with our speech recognition service for non-English.
